doc/data-mining-library/source/tutorial/data.rst

```diff
@@ -60,6 +60,59 @@ The following script wraps-up everything we have done so far and lists first 5 d
 
 Note that data is an object that holds both the data and information on the domain. We show above how to access attribute and class names, but there is much more information there, including that on feature type, set of values for categorical features, and other.
 
+Creating a Data Table
+---------------------
+
+To create a data table from scratch, one needs two things, a domain and the data. The domain is the description of the variables, i.e. column names, types, roles, etc.
+
+First, we create the said domain. We will create three types of variable, numeric (ContiniousVariable), categorical (DiscreteVariable) and text (StringVariable). Numeric and categorical variables will be used a features (also known as X), while the text variable will be used as a meta variable.
+
+    >>> from Orange.data import Domain, ContinuousVariable, DiscreteVariable, StringVariable
+    >>> 
+    >>> domain = Domain([ContinuousVariable("var1"), DiscreteVariable("var2", values=["red", "blue"])], metas=[StringVariable("var3")])
+
+Now, we will build the data with numpy.
+
+    >>> import numpy as np
+    >>> 
+    >>> var1 = np.array([1.2, 1.4, 1.5, 1.1, 1.2])
```

Comment:
I'm thinking - would it be better to call this `column1` or `col1`? This is not a variable.

Reply:
True, will fix it.